### PR TITLE
migrate_manual_merged_user

### PR DESCRIPTION
**Pull request by chennan47**
<b>Purpose</b>
Add migrate script to migrate manually merged user. Fix issue https://github.com/CenterForOpenScience/osf.io/issues/2470

### CODE REVIEW

File: tests/factories.py [hunk 1]
```diff
@@ -82,6 +82,9 @@ class UserFactory(ModularOdmFactory):
     is_claimed = True
     api_keys = []
     date_confirmed = datetime.datetime(2014, 2, 21)
+    merged_by = None
+    email_verification = {}
```
Comment (sloria) [inline tests/factories.py]: Should be `email_verifications`.

Comment (chennan47) [inline tests/factories.py]: fixed in 741bf7a

File: scripts/migrate_manual_merged_user.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            user.password,
+            user.email_verifications,
+            user.verification_key
+        )
+    )
+
+
+def main():
+    init_app(routes=False)  # Sets the storage backends on all models
+    if 'dry' in sys.argv:
```
Comment (sloria) [inline scripts/migrate_manual_merged_user.py]: Why not just pass a `dry` parameter to `do_migration` and only save if `dry=False`?

**Comment by sloria** [comment]
Pass finished.

**Comment by chennan47** [comment]
@sloria rdy for another pass

**Comment by sloria** [comment]
@lyndsysimon Can you verify the correctness of this migration?

**Comment by lyndsysimon** [comment]
@sloria I've reviewed and verify that it is the correct course of action.

**Pull request merged**